src/index.js

```diff
@@ -56,6 +56,8 @@ module.exports = {
   methods: {
     $resolve(name) {
       if (!this.$queueResolved[name]) {
+        // Adding QueueScheduler to support delayed jobs as described here https://docs.bullmq.io/guide/jobs/delayed
+        new QueueScheduler(name, { connection: this.$connection })
```

Comment:
Could you just store this into some kind of `this.$queueDelay[name]` ? (With the addition of `this.$queueDelay = {}` into the `created()`)
I'm afraid this could be remove/free by the V8 garbage collector